examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -39,6 +39,14 @@
           {
             "name": "aad-pod-identity",
             "enabled": true
+          },
+          {
+            "name": "coredns",
+            "enabled": true,
+            "config": {
+              "min-replicas": "1",
+              "nodes-per-replica": "10"
```

Comment:
This configuration in concert with the overall node count in the cluster, so that when we perform node scale out E2E tests we are also able to reliably test coredns-autoscaler behaviors.